src/components/LessonList/components/PaginatedLessonList/components/LessonTeaser/index.js

```diff
@@ -35,14 +34,14 @@ export default ({instructor, lesson}) => {
         }}>
           {lesson.summary
             ? <Markdown>
-                {truncate(lesson.summary, {length: 144})}
+                {lesson.summary}
               </Markdown>
-            : '...'
+            : null
           }
         </div>
         <div className='mt2'>
           <Link to={`/lessons/${lesson.slug}`}>
-            <Button size='small'>
+            <Button size='extra-small'>
```

Comment:
We need to make sure that stuff looks nice and not just go with the default because it's the default. It's situational and we should all use best judgement or ask when in doubt.

`egghead-ui` is hopefully going to become a rapid prototyping toolkit for egghead software. Sensible defaults that "just work" and look nice would be rad.